unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendRequestsTabComponentView.cs

```diff
@@ -251,6 +259,9 @@ private Pool GetEntryPool()
 
     private void SendFriendRequest(string friendUserName)
     {
+        friendUserName = friendUserName.Trim()
+            .Replace("\n", "")
```

Comment:
just for a more safe usage could you change `\n` with `System.Environment.NewLine`?

Reply:
i would prefer to maintain it the way it is now, since for non-unix platforms `System.Environment.NewLine` is `\n\r`.. if you write just `\n` in the search entry, it will not be removed

Reply:
Makes more sense yeah